.github/actions/build-cache/action.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Set up build cache'
+description: 'Sets up caching for KDF builds'
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Set up rust-cache
+      uses: Swatinem/rust-cache@v2
```

Comment:
Do we not need to set 'key' like in the previous action.yml?

Reply:
No, we don't need to.